org.jacoco.core/src/org/jacoco/core/JaCoCo.java

```diff
@@ -19,9 +19,17 @@
  */
 public final class JaCoCo {
 
-	/** Qualified build version of the JaCoCo core library. */
+	/** Version of JaCoCo core. */
```

Comment:
Ups my bad. I was wrongly assuming we have the plain version here. From my point of view the plain version would be enough here. What do you think, can we change this to `project.version`?

If we don't want to change this in this PR the qualified version is also good for me.

Reply:
> What do you think, can we change this to `project.version`?

Well I have mixed feelings:

Maybe not a big deal to change it, however strictly speaking it is part of our public API on it's own and a return value of Agent API `org.jacoco.agent.rt.IAgent.getVersion`, also it is shown by `version` command in CLI and always displayed in footer (right-bottom corner) of html reports.

And on one side, since qualifier is build timestamp, maybe nice that people will see how old version they are using. On the other side, since their configuration files use unqualified version, presence of qualifier might be confusing.

We can introduce `VERSION_SHORT` by analogy with `COMMITID_SHORT`, but maybe this is an overkill.

> If we don't want to change this in this PR the qualified version is also good for me.

So let's keep it as is for now.

> I was wrongly assuming we have the plain version here.

I'll add back word "Qualified" so that it can again remind us next time :wink: